drivers/sensor/tdk/icm42x70/icm42x70.c

```diff
@@ -829,57 +854,60 @@ static int icm42670_fetch_from_registers(const struct device *dev, enum sensor_c
 }
 #endif
 
-static int icm42670_sample_fetch(const struct device *dev, enum sensor_channel chan)
+static int icm42x70_sample_fetch(const struct device *dev, enum sensor_channel chan)
 {
 	int status = -ENOTSUP;
 
-	icm42670_lock(dev);
+	icm42x70_lock(dev);
 
 #ifdef CONFIG_TDK_APEX
 	if ((enum sensor_channel_tdk_apex)chan == SENSOR_CHAN_APEX_MOTION) {
-		status = icm42670_apex_fetch_from_dmp(dev);
+		status = icm42x70_apex_fetch_from_dmp(dev);
 	}
 #endif
 
 	if ((chan == SENSOR_CHAN_ALL) || (chan == SENSOR_CHAN_ACCEL_XYZ) ||
 	    (chan == SENSOR_CHAN_ACCEL_X) || (chan == SENSOR_CHAN_ACCEL_Y) ||
-	    (chan == SENSOR_CHAN_ACCEL_Z) || (chan == SENSOR_CHAN_GYRO_XYZ) ||
-	    (chan == SENSOR_CHAN_GYRO_X) || (chan == SENSOR_CHAN_GYRO_Y) ||
-	    (chan == SENSOR_CHAN_GYRO_Z) || (chan == SENSOR_CHAN_DIE_TEMP)) {
-#ifdef CONFIG_ICM42670_TRIGGER
-		status = icm42670_fetch_from_fifo(dev);
+	    (chan == SENSOR_CHAN_ACCEL_Z)
```

Comment:
nit: prefer `sensor.h`'s `SENSOR_CHANNEL_IS_ACCEL(chan)` and `SENSOR_CHANNEL_IS_GYRO(chan)` for readability.